db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/FromBaseTable.java

```diff
@@ -240,7 +239,7 @@
 
     private ValueNode pastTxIdExpression = null;
 
-    private long pastTxnId = -1;
+    private long pastTxnId = TransactionController.TIME_TRAVEL_UNSET;
```

Comment:
Is this correct? We were setting it to `-1` (`OLDEST`) and now it's being set to `-2` (`UNSET`)

Reply:
Yes I think it is, `pastTxnId` is set accordingly later on in `bindNonVTITables`.